### ISSUE TITLE
mArchiveList returning different results for Montage 3.3 and 4.0

### ISSUE THREAD

**Issue by astrofrog** [Caltech-IPAC/Montage#8]
For the following command:

```
$ mArchiveList 2MASS k M31 0.2 0.2 m31_2mass_k_4.0.tbl
```

Montage 3.3 returns a list of 8 images, while 4.0 returns a list of 16 images. Is this normal?

**Comment by JohnGood** [comment]
The change wasn't in mArchiveList but in the underlying web service.  The older version of the service used a spatial index for finding images based on point locations (the center of the images).  To find all the images overlapping a region, the region was padded by the diagonal size of a 2MASS image.  This is equivalent to having the images be circular (bounding circle) regions.

Some months ago, we updated the web service to use an R-Tree index (based on a new Montage module).  The images could then be real polygon regions on the sky.  However, we forgot to take the padding out of the search region calculation, so we have been getting a slightly-too-large return.  Since in most cases we use a tool like mSubset to trim the list down exactly,
the larger list doesn't have too much an effect but it could cause some people to download images they don't need.

The padding has now been removed and in this case the service reduces the count even further since a couple of those eight matched with the bounding circle but not an exact polygon check.
